src/components/Screen/index.js

```diff
@@ -94,13 +94,6 @@ class ScreenHeader extends Component {
 							}
 						</Header.Action>
 					</Tooltip.Trigger>
-					{(!expanded && !windowed) && (
-						<Tooltip.Trigger content={I18n.t('Expand chat')} placement='bottom-left'>
-							<Header.Action aria-label={I18n.t('Expand chat')} onClick={onOpenWindow}>
-								<OpenWindowIcon width={20} height={20} />
-							</Header.Action>
-						</Tooltip.Trigger>
-					)}
```

Comment:
@ear-dev We should comment it instead of removing. Will help in future if we want to reenable it.

Reply:
@Shailesh351 Initially I tried commenting it out, but I could not get it to work.  As if comments are not allowed in this kind of html block?  Let me know if you have a suggestion.  Thanks.

Reply:
@ear-dev We can actually do like below,
```
{/* Visasat : Hide Expand Button */}
{/* {(!expanded && !windowed) && (
	<Tooltip.Trigger content={I18n.t('Expand chat')} placement='bottom-left'>
		<Header.Action aria-label={I18n.t('Expand chat')} onClick={onOpenWindow}>
			<OpenWindowIcon width={20} height={20} />
		</Header.Action>
	</Tooltip.Trigger>
)} */}
```